localization/yabloc/yabloc_monitor/README.md

```diff
@@ -0,0 +1,27 @@
+# yabloc_monitor
+
+YabLoc monitor is a node that monitors the status of the YabLoc localization system. It is a wrapper node that monitors the status of the YabLoc localization system and publishes the status as diagnostics.
+
+## Feature
+
+### Availability
+
+The node monitors the final output pose of YabLoc to verify the availability of YabLoc.
+
+### Others
+
+To be added,
+
+## Interfaces
+
+### Input
+
+| Name           | Type                        | Description                     |
+| -------------- | --------------------------- | ------------------------------- |
+| `/yabloc/pose` | `geometry_msgs/PoseStamped` | The final output pose of YabLoc |
```

Comment:
availability_module.cpp subscribes `/yabloc_pose` rather than `/yabloc/pose` by default.
```suggestion
| `/yabloc_pose` | `geometry_msgs/PoseStamped` | The final output pose of YabLoc |
```

Reply:
Right, it was wrong.
Fixed: https://github.com/autowarefoundation/autoware.universe/pull/4395/commits/fd68effceb8e8aa297a3b86e0349817f7bab1011